pop-up.js

```diff
@@ -0,0 +1,61 @@
+const projects = [
+  {
+    name: 'Tonic',
+    frame: './images/fame2.png',
+    cover: './images/popUp.png',
+    textPopUp: 'Lorem ipsum dolor sit amet  Lorem ipsum dolor, sit amet consectetur adipisicing elit. Eligendi voluptatibus consequatur iure consequuntur odit, ex ipsa debitis. Reiciendis veniam quae corporis non quis voluptatem nam maiores? Molestiae facere praesentium laborum?',
+    technologies: ['HTML', 'CSS', 'JavaScript'],
+    seeLive: 'https://example.com',
+    seeSource: 'https://github.com/migcm06/miguel-portfolio.git',
+  },
+  {
+    name: 'Multi-Post',
+    frame: './images/fame2.png',
+    cover: './images/popUp.png',
+    textPopUp: 'Lorem ipsum dolor sit amet  Lorem ipsum dolor, sit amet consectetur adipisicing elit. Eligendi voluptatibus consequatur iure consequuntur odit, ex ipsa debitis. Reiciendis veniam quae corporis non quis voluptatem nam maiores? Molestiae facere praesentium laborum?',
+    technologies: ['HTML', 'CSS', 'JavaScript', 'Query'],
+    seeLive: 'https://example.com',
+    seeSource: 'https://github.com/migcm06/miguel-portfolio.git',
+  },
+];
+/* eslint-disable no-unused-vars */
+function openPopup(index) {
+  const overlay = document.querySelector('.overlay');
+  const popUpContainer = document.querySelector('.pop-up-container');
+  const popUp = document.querySelector('.pop-up');
+  const project = projects[index];
+  const buttons = document.querySelector('.button-project');
+
+  const popupHTML = `
+  <div id="close-pop"><i class="bi bi-x"></i></div>
+    <h2 class="title-pop-up">${project.name}</h2>
+    <img src="${project.frame}" class="frame-pop">
+    <div class="pop-up-img">
+      <img
+        src="${project.cover}"
+        alt="Background image"
+        class="pop-up-image desktop"
+      />
+    </div>
+    <div class="text-pop-up">${project.textPopUp}</div>
+    <div class="right-links">
+    <div class="tag-pop-up">${project.technologies.map((tech) => `<a href="#" class="project-tech">${tech}</a>`).join('')}</div>
+    <div class="buttons-pop">
+    <a class="btn-pop" href="${project.seeLive}" target="_blank">See Live <img src="./images/Export.svg" class="icon-pop"></a>
+    <a class="btn-pop" href="${project.seeSource}" target="_blank">See Source <img src="./images/Github.svg" class="icon-pop"></a>
+  </div>
+  </div>
+  `;
```

Comment:
- [ ] kindly consider sticking to the Figma design as much as possible on both desktop and mobile screens  to make your design an exact duplicate of the given one  check the images below to see what to improve 
- Figma desktop design 👇 
<img width="677" alt="Screen Shot 2023-07-26 at 17 25 37" src="https://github.com/migcm06/miguel-portfolio/assets/60146030/f6049002-f330-4ab8-852a-dc68fa944272">

- your desktop design 👇 
<img width="1423" alt="Screen Shot 2023-07-26 at 17 26 45" src="https://github.com/migcm06/miguel-portfolio/assets/60146030/4c566c7d-de0b-4f06-8f81-24c132cd5c36">


- Figma mobile design 👇 
<img width="319" alt="Screen Shot 2023-07-26 at 17 30 41" src="https://github.com/migcm06/miguel-portfolio/assets/60146030/e516ee7a-207c-4cb5-9193-4a701e2f2785">

- your mobile design 👇 
<img width="369" alt="Screen Shot 2023-07-26 at 17 29 23" src="https://github.com/migcm06/miguel-portfolio/assets/60146030/9fb3b509-f93d-4679-b555-5b5f4f9c7e07">